ignite/services/network/networkchain/prepare.go

```diff
@@ -253,6 +257,22 @@ func (c Chain) applyGenesisValidators(ctx context.Context, genesisVals []network
 	return c.updateConfigFromGenesisValidators(genesisVals)
 }
 
+// applyParamChanges adds the param changes into the genesis
+func applyParamChanges(
+	genesis *cosmosgenesis.Genesis,
+	paramChanges []networktypes.ParamChange,
+) error {
+	for _, pc := range paramChanges {
+		if err := genesis.Update(
+			jsonfile.WithKeyValueByte(cosmosgenesis.ParamField(pc.Module, pc.Param), pc.Value),
+		); err != nil {
+			return fmt.Errorf("genesis cannot be updated: %w", err)
```

Comment:
What about adding the details of the param in the error label ? 

Because currently, if an error occurs, the label will be 

```
error applying param changes to genesis: genesis cannot be updated: a json error`
```

While it could be 

```
error applying param changes to genesis: param "x" value "v" cannot be updated: a json error`
```

Reply:
https://github.com/ignite/cli/pull/3208/commits/429b8ad3986e8c22560fdad341682d195ac84b3e